src/pattern-library/components/patterns/prototype/LMSContentSelectionPage.tsx

```diff
@@ -0,0 +1,758 @@
+                    <div className="space-y-1.5 pt-1">
+                      <div className="leading-none text-[14px] text-end font-medium text-slate-600 uppercase">
+                        Assignment content
+                      </div>
+                      <div className="text-[14px] text-end leading-none font-normal text-stone-500">
+                        <p>Select content for your assignment</p>
+                      </div>
+                    </div>
+                    <div className="space-y-6 text-[14px] leading-none">
+                      <div className="flex flex-row p-y-2">
+                        <div className="grid grid-cols-2 gap-2 w-full">
+                          <Button5 contentType="web page | PDF" icon={LinkIcon}>
+                            URL
+                          </Button5>
+                          <Button5>Canvas</Button5>
+                          <Button5>Google Drive</Button5>
+                          <Button5 contentType="article">JSTOR</Button5>
+                          <Button5 icon={OneDriveIcon}>OneDrive</Button5>
+                          <Button5 contentType="book" icon={VitalsourceIcon}>
+                            VitalSource
+                          </Button5>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </Card>
+            </Library.Demo>
+          </Library.Example>
+
+          <Library.Example title="Tiles">
+            <p>
+              This sketch demonstrates a possible tile layout, with a{' '}
+              {'"selected"'} styling (and hover).
+            </p>
+
+            <Library.Demo>
+              <Card width="custom" classes="w-[700px]">
+                <CardHeader variant="secondary" title="Assignment details" />
+                <div className="px-3 py-6">
+                  <div className="grid grid-cols-[8em_1fr] gap-x-6 gap-y-3">
+                    <div className="space-y-1.5 pt-1">
+                      <div className="leading-none text-[14px] text-end font-medium text-slate-600 uppercase">
+                        Content
+                      </div>
+                      <div className="text-[14px] text-end leading-none font-normal text-stone-500">
+                        <p>Select content for your assignment</p>
+                      </div>
+                    </div>
+                    <div className="space-y-6 text-[14px] leading-none">
+                      <div className="flex flex-row p-y-2">
+                        <div className="grid grid-cols-4 gap-2 w-full">
+                          <Button6 contentType="web page | PDF" icon={LinkIcon}>
+                            URL
+                          </Button6>
+                          <Button6>Canvas</Button6>
+                          <Button6>Google Drive</Button6>
+                          <Button6 contentType="article">JSTOR</Button6>
```

Comment:
Should this button use the JSTOR icon?

Reply:
Yes, but finding a workable JSTOR logo in SVG format was eluding me and I didn't want to drop more time into it. Their more elegant logo, which I was able to find in (COMPLEX) SVG  is not workable as an icon:

![image](https://user-images.githubusercontent.com/439947/235143373-5b4e96cc-bf26-4304-8ea8-1ab717f8b0a5.png)

There is a variant I've seen that is just a "J", but I could not find an SVG of it:

![image](https://user-images.githubusercontent.com/439947/235143513-55649532-b56b-4fb6-a5be-b679d5105b63.png)

I was able to find Google Drive, OneDrive and, surprisingly, a VitalSource image in SVG format, which allowed me to drop them in quickly here. 

I can add a callout to the affected sketches to alert anyone else who might look at them about this.

Reply:
Oh, derp, you probably noticed that I did try to create an icon from the elaborate logo and it's in the library but I didn't use it. It didn't work at small sizes. I'll remove the `JSTORLogoIcon` component.